third_party/lasread/lidario.go

```diff
@@ -385,8 +385,9 @@ func (las *LasFile) read() error {
 func (las *LasFile) readHeader() error {
 	las.Lock()
 	defer las.Unlock()
-	b := make([]byte, 243)
-	if _, err := las.f.ReadAt(b[0:243], 0); err != nil && err != io.EOF {
+	// For 1.4 set max length
+	b := make([]byte, 375)
```

Comment:
While this should work for LAS 1.4 I suspect we might create compatibility problems with LAS 1.2. Should we first read the first 26 bytes up to the VersionMajor and VersionMinor and then decide how much more to read to fill the headers?

Reply:
I have tried both 1.2 and 1.4, but it is also ok to read the first 26 bytes to judge.I can check it later. Thank you  :)

Reply:
![image](https://user-images.githubusercontent.com/2793680/174537339-3d9c7d61-b010-47d1-82e4-4afa684fb18c.png)
----
this is las1.3 header, the total length is 235